app/client/src/features/events/EventDetailsCard/EventDetailsCard.tsx

```diff
@@ -34,36 +28,23 @@ export const EVENT_DETAILS_CARD_FRAGMENT = graphql`
 `;
 
 interface Props {
-    className?: string;
     fragmentRef: EventDetailsCardFragment$key;
 }
 
-export function EventDetailsCard({ className, fragmentRef }: Props) {
+export function EventDetailsCard({ fragmentRef }: Props) {
     const { topic, title, description } = useFragment(EVENT_DETAILS_CARD_FRAGMENT, fragmentRef);
     const classes = useStyles();
-    const [isIn, setIsIn] = React.useState(false);
 
     return (
-        <div className={clsx(classes.root, className)}>
```

Comment:
Is there a specific reason we're remove the option to override properties via className?

Reply:
I think I just removed it since I replaced the div component with a Grid component. Though, I can add it back in and have it applied to the Grid component instead.

Reply:
Ah, I see. I would usually err on the side of not removing extendability, however this component is only used in one place.  It's also pretty easy to add it back in.  So don't worry about it for now.

Reply:
Got it.